src/Appium.Net/Appium/AppiumCommand.cs

```diff
@@ -118,9 +104,6 @@ public class AppiumCommand
                 "/session/{sessionId}/appium/device/terminate_app"),
             new AppiumCommand(HttpCommandInfo.PostCommand, AppiumDriverCommand.IsAppInstalled,
                 "/session/{sessionId}/appium/device/app_installed"),
-            // TODO: Use mobile:queryAppState
-            new AppiumCommand(HttpCommandInfo.PostCommand, AppiumDriverCommand.GetAppState,
```

Comment:
mac2 driver has `macos:queryAppState`, but it doens't have this endpoint so this PR define this method only in android/ios